ProcessList.c

```diff
@@ -364,4 +537,14 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
          p->updated = false;
       }
    }
+
+   // clean up hashtable in order to not have
+   // processes from privious build in it
+   //
+   // sorting algorithm relayes on the fact that
+   // len(this->displayTreeSet) == len(this->processes)
+   Hashtable_delete(this->displayTreeSet);
+   this->displayTreeSet = Hashtable_new(4096, false);
```

Comment:
Maybe introduce `Hashtable_clear()` instead of delete + new 

```c
void Hashtable_clear(Hashtable* this) {
   for (int i = 0; i < this->size; i++) {
      if (this->owner) {
            free(this->buckets[i].value);
      this->buckets[i].value = NULL;
   }
   this->items = 0;
}
```

Reply:
`Hashtable_delete` does delete the buckets (and `Hashtable_new` creates them). `Hashtable_clear` would not deallocate and allocate them.

Reply:
It makes sense

Reply:
Upstreamed to master.